eqc/tcp_proto_eqc.erl

```diff
@@ -244,3 +247,33 @@ prop_encode_decode_metrics() ->
                 L2 = lists:sort(Rev),
                 L1 == L2
             end).
+
+prop_encode_decode_bucket_info() ->
+    ?FORALL(Msg = {Res, PPF, TTL}, {resolution(), pos_int(), ttl()},
+            case valid_time(Res) of
```

Comment:
valid time is misleading here, `infinity` isn't a valid time but it's a valid ttl.